lang/simplified_chinese.lng

```diff
@@ -0,0 +1,748 @@
+STR_LINES_USED:{BLACK}使用线路：{STRING}
+
+# *Metro operators
+# TODO Remove City name, as company has it, add more relavant info like in private stuff
+STR_TOKYOMETRO:{LTBLUE}东京地铁
+STR_TOEI:{GREEN}都营地铁
+STR_YOKOHAMA:{GREEN}横滨市交通局
+STR_KITAKYU:{RED}北大阪急行铁道
+STR_OSAKAMETRO:{LTBLUE}大阪地铁
+STR_KYOTO:{GREEN}京都市交通局
+STR_NAGOYA:{LTBLUE}名古屋市交通局
+STR_SENDAI:{BLUE}仙台市交通局
+STR_KOBE:{LTBLUE}神户市交通局
+STR_FUKUOKA:{LTBLUE}福冈市交通局
+
+# *Private railways
+# Tokyo
+STR_TOKYU:{ORANGE}东急公司
+STR_TOBU:{BLUE}东武铁路
+STR_SEIBU:{LTBLUE}西武铁路
+STR_ODAKYU:{BLUE}小田急电铁
+STR_SOTETSU:{PURPLE}相模铁道（相铁）
+STR_KEIO:{RED}京王公司
+STR_KEIKYU:{RED}京急公司
+STR_KEISEI:{LTBLUE}京成电铁
+
+STR_TWR:{LTBLUE}东京滨区快速交通
+STR_TSUKUBA:{BLUE}筑波快线
+STR_SHIZUTETSU:{RED}静冈铁道
+STR_IZUHAKONE:{GREEN}伊豆箱根铁道
+STR_CHICHIBU:{LTBLUE}秩父铁道
+STR_SHIBAYAMA:{GREEN}柴山铁道
+STR_MINATO:{BLUE}横滨港未来铁道
+STR_SHIN_KEISEI:{PURPLE}新京成电铁
+STR_HOKUSO:{YELLOW}北総铁道
+STR_CHIBA_NT:{LTBLUE}千叶新都市铁道
+STR_CHIBA_KYUKO:{GREEN}千叶急行电铁
+
+# Osaka
+STR_KINTETSU:{BLUE}近畿铁道
+STR_HANKYU:{PURPLE}阪急铁道
+
+# *Metro lines
+STR_TOKYO_GINZA:{ORANGE}银座线
+STR_TOKYO_MARU:{RED}丸之内线
+STR_TOKYO_MARU_BR:{RED}丸之内支线
+STR_TOKYO_HIBIYA:{SILVER}日比谷线{TINYFONT}  途经东武线
+STR_TOKYO_TOZAI:{LTBLUE}东西线{SILVER}{TINYFONT}  途经JR中央-総武线
+STR_TOKYO_TOZAI_TOYO:{GREEN}东叶高速线{BLACK}与{LTBLUE}东西线
+STR_TOKYO_CHIYODA:{GREEN}千代田线{SILVER}{TINYFONT}  途经JR常磐线 与小田急线
+STR_TOKYO_CHIYODA_BR:{GREEN}千代田支线
+STR_TOKYO_YURA:{CREAM}有乐町线{SILVER}{TINYFONT}  途经东武 与西武线
+STR_TOKYO_YURA_FUKU:{CREAM}有乐町线{BLACK}与{BROWN}福新线{SILVER}{TINYFONT}  途经东武 与西武线
+STR_TOKYO_HANZO:{PURPLE}半藏门线{SILVER}{TINYFONT}  途经东急 与东武线
+STR_TOKYO_NAMBOKU:{LTBLUE}南北线{SILVER}{TINYFONT}  途经东急 与相铁线
+STR_TOKYO_NAMBOKU_SAITAMA:{LTBLUE}埼京快速线{BLACK}与{LTBLUE}南北线
+STR_TOKYO_ASAKUSA:{RED}浅草线{SILVER}{TINYFONT}  途经京急与京成线
+STR_TOKYO_MITA:{BLUE}三田线{SILVER}{TINYFONT}  途经东急与相铁线
+STR_TOKYO_SHINJUKU:{GREEN}新宿线{SILVER}{TINYFONT}  途经京王线
+STR_TOKYO_OEDO:{RED}大江户线
+STR_YOKOHAMA_BLUE:{BLUE}蓝线
+STR_YOKOHAMA_GREEN:{GREEN}绿线
+STR_OSAKA_MIDO_NAMBOKU:{RED}御堂筋线{BLACK}与{RED}南北线
+STR_OSAKA_MIDO:{RED}御堂筋线
+STR_OSAKA_MIDO_YOTSU:{RED}御堂筋线{BLACK}与{BLUE}四桥线
+STR_OSAKA_YOTSU:{BLUE}四桥线
+STR_OSAKA_SENNICHI:{RED}千日前线
+STR_OSAKA_TANIMACHI:{PURPLE}谷町线
+STR_OSAKA_CHUO:{GREEN}中央线
+STR_OSAKA_KEIHANNA:{GREEN}京阪线
+STR_OSAKA_SAKAISUJI:{BROWN}堺筋线
+STR_OSAKA_TSURUMI:{GREEN}长堀鹤见绿地线
+STR_OSAKA_IMAZATO:{ORANGE}今里筋线
+STR_KYOTO_KARASUMA:{GREEN}烏丸线
+STR_KYOTO_TOZAI:{ORANGE}东西线
+STR_NAGOYA_HIGASHI:{ORANGE}东山线
+STR_NAGOYA_MEIJO:{PURPLE}名城线
+STR_NAGOYA_TSURUMIRAI:{LTBLUE}鹤未来线{SILVER}{TINYFONT}  途经名铁线
+STR_NAGOYA_SAKURA:{RED}樱线
+STR_NAGOYA_KAMIIDA:{RED}上飯田线{SILVER}{TINYFONT}  途经名铁线
+STR_SENDAI_NAMBOKU:{GREEN}南北线
+STR_SENDAI_TOZAI:{LTBLUE}东西线
+STR_KOBE_SEISHIN:{GREEN}西神山手线{BLACK}与{BROWN}北神线
+STR_KOBE_KAIGAN:{LTBLUE}海岸线
+STR_FUKUOKA_KUKO:{ORANGE}空港{BLACK}与{LTBLUE}箱崎线
+STR_FUKUOKA_NANAKUMA:{GREEN}七隈线
+
+# mu_cars
+MUCAR_1435_M:JP+ 私铁动车组车厢（第三轨）
+MUCAR_1435:JP+ 私铁动车组车厢（1435/1372㎜）
+MUCAR_1067:JP+ 私铁动车组车厢（1067㎜）
+MUCAR_DESC:{BLUE}用于替换{GREEN}列车{BLUE}编组中的{GREEN}动车和拖车{BLUE}。
+
+# *Train Prefixes
+STR_NAME_EIDAN:伊达快速
```

Comment:
I understand the importance of keeping the information list as clear as possible; however, here's the situation:

People who use Chinese to play oftentimes don't have a well developed information basis on Japanese railway companies. Am example would be JNR. Of course it's translation could be "国鉄" in Japanese, but I cannot translate that to "国铁" in Chinese, because the "国" here is referring to "Japan," and thus I translated it to "日本国有铁道." Sometimes keeping some verbose messages is for the player to understand the concept more easily.

The "eidan" here is a mistake. I looked up the wrong site and got the wrong translation. I will correct this in the future.

Reply:
By the way, for those liveries that has a Japanese name which is translated from English (new red arrow, for example), do you think I should also keep the Japanese name in the translation file as a reference? Or only the English name? Or no reference at all? Or just keep the original name?

Reply:
Oh it was just an advice, you shall translate as you see it fits
As for the liveries names, I think it would be better to leave them as they are on English. Especially since their original names usually come from English words.